components/demo/demo-page.js

```diff
@@ -0,0 +1,13 @@
+import './demo-snippet.js';
+import './code-view.js';
+import '../colors/colors.js';
+import '../typography/typography.js';
+
+document.body.classList.add('d2l-typography');
+
+export const showPage = async() => {
+	if (document.fonts) {
+		await document.fonts.ready;
+	}
+	document.body.removeAttribute('unresolved');
+};
```

Comment:
The old code (that came with Polymer) waited for the `WebComponentsReady` event to fire... I wonder if it's still getting fired by the loader code and we could do the same?

Reply:
That was the first thing I tried.  WCR works sufficiently in Chrome and Safari, however WCR fires before the module script/imports in FF, Edge, and IE11.  :(